math-doc/math/scribblings/math-matrix.scrbl

```diff
@@ -525,8 +525,26 @@ in list must have the same number of columns.
                  (matrix-stack (list M0 M1 M2))]
 }
 
+@deftogether[(@defproc[(matrix-set-col [M (Matrix A)] [new-col (Matrix A)] [idx Integer]) (Matrix A)]
+              @defproc[(matrix-set-row [M (Matrix A)] [new-row (Matrix A)] [idx Integer]) (Matrix A)])]{
```

Comment:
The abbreviation `col` for column is used throught the matrix library, so it makes sense to call it `matrix-set-col`, but I think you use `matrix-set-column` where you define it.

The order of the arguments needs to be: `(matrix-set-col M idx new-col)` in order to be consistent
with other functions (compare `(vector-set! v i new-value)`.

Same goes for `matrix-set-row`.